addons/test/misra-test.c

```diff
@@ -13,6 +13,8 @@ typedef unsigned int       u32;
 typedef unsigned long long u64;
 
 //// 3.1
+extern int a123456789012345678901234567890x;
+extern int a123456789012345678901234567890y; //5.1
```

Comment:
I have added it here

Reply:
yes I saw that but when I will look for the tests in the future I will search for "misra_5_1" in my text editor.

Reply:
it is not entirely clear what these 3 lines are testing:

    //// 3.1
    +extern int a123456789012345678901234567890x;
    +extern int a123456789012345678901234567890y; //5.1

when I read it in the future I will probably think it looks like the primary purpose of these 3 lines are probably to test rule 3.1... and then it will seem that the 2 lines below the comment looks wrong and should be removed.